tools/osbuild-mpp

```diff
@@ -509,6 +511,50 @@ class Image:
         return cls(size, table)
 
 
+class LocalFile:
+    def __init__(self, base64_content, sha256, sha512):
+        self.base64_content = base64_content
+        self.sha256 = sha256
+        self.sha512 = sha512
+
+    @property
+    def digest(self):
+        return f"sha256:{self.sha256}"
+
+    @classmethod
+    def from_dict(cls, js):
+        path = js["path"]
+        with open(path, 'rb') as stream:
+            data = stream.read()
```

Comment:
Maybe we want some kind of nice error message here rather than just throwing an exception?

Reply:
Adjusted :)